src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -18,6 +18,10 @@
     <NuGetOutputDirectory>&quot;$(PackagesOutDir.TrimEnd('\'))&quot;</NuGetOutputDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <LocalPublishPropsFile Condition="'$(LocalPublishPropsFile)'==''">$(SourceDir)\localpublish.props</LocalPublishPropsFile>
```

Comment:
I will make this $(PackagesDir) and will update the CoreCLR repo to have this variable